ios/RCTWebRTC/WebRTCModule+Permissions.m

```diff
@@ -23,13 +23,15 @@ - (AVMediaType)avMediaType:(NSString *)mediaType {
                   : (NSString *)mediaType resolver
                   : (RCTPromiseResolveBlock)resolve rejecter
                   : (RCTPromiseRejectBlock)reject) {
+#if TARGET_OS_TV
+    return;
```

Comment:
You need to resolve the promise before returning.

Reply:
This is the way you are meaning?

Reply:
yep!